src/org/exist/storage/NativeBroker.java

```diff
@@ -1205,6 +1213,10 @@ private void copyModeAndAcl(final Permission srcPermission, final Permission des
         }
     }
 
+    private boolean isSubCollection(final Collection col, final Collection sub) {
+        return sub.getURI().startsWith(col.getURI());
```

Comment:
yes, each collection is sub collection of `/db` and that's why `/db` can't be moved.